spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -1063,7 +1069,8 @@ def create_new_client(self, give_focus=True, filename='', is_cython=False,
                               show_elapsed_time=show_elapsed_time,
                               reset_warning=reset_warning,
                               given_name=given_name,
-                              ask_before_restart=ask_before_restart)
+                              ask_before_restart=ask_before_restart,
+                              css_path=self.css_path)

```

Comment:
Please look for other instances of `ClientWidget` here and pass `self.css_path` to them too (I'm almost sure this is not the only one.)